components/ai_chat/resources/page/components/svg/caret.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import * as React from "react"
+
+const SvgComponent = (props: any) => (
```

Comment:
should the type be svg props?

Reply:
or css class? Or do we even need props? Doesn't look like they're used

Reply:
removed props